docs/source/10-minutes-to-dask.rst

```diff
@@ -373,6 +373,7 @@ triggering computation, we can inspect the task graph to figure out what's going
          >>> result.visualize()
 
       .. image:: images/10_minutes_dataframe_graph.png
+         :alt: Rendered task graph with several nodes connected by arrows, dependent on one of the input blocks, with node names like "cumsum" and "getitem".
```

Comment:
```suggestion
         :alt: Dask task graph with several nodes connected by arrows, dependent on one of the input blocks, with node names like "cumsum" and "getitem".
```

This suggestion is possibly excessively nit-picky. I'm not sure that the word "rendered" really adds anything here (all the images are rendered). I think it's more important for text readers to say what ithe image is of, eg: a Dask task graph.